### ISSUE TITLE
Change default fonts to Materials Icons

### ISSUE THREAD

**Issue by andypl78** [nextcloud/server#2730]
Can I ask you to change fonts for Materials Icons?
The problem is with the proper display of fonts Central Europe, for example, Polish
I put on my server Material Icons and there are no problems with fonts
If the need for amendments can assist ;)

**Comment by MorrisJobke** [comment]
cc @jancborchardt

**Comment by jancborchardt** [comment]
@andypl78 sorry, I don’t fully understand what you mean. Can you maybe rephrase the issue? Do you mean the Material Design icons, or do you have a screenshot of what you mean?

For info: Nextcloud has standard icons, and we use SVG for icons since they are scalable and modifiable. Icon fonts are and were always a temporary hack (which we never used). :)
